### DISCUSSION TITLE
I need to know what email is on a Supabase Project

### DISCUSSION THREAD

**Discussion by XolifyDev** [supabase/supabase#20206] [Questions]
Hey! I wanted to know if someone could help me find out what email is connected to a project. I only know the project id. I kinda forgot what email I used to login 😭.

**Comment by GaryAustin1** [comment]
Normally people have signed in with Github and the email inputs do not apply.
If you really did signin with email directly and have no idea, the only thing I can think of is to contact support with a list of emails.... but I suspect you would have to prove each one you have access to.

**Reply by XolifyDev** [reply]
> Normally people have signed in with Github and the email inputs do not apply.
> 
> If you really did signin with email directly and have no idea, the only thing I can think of is to contact support with a list of emails.... but I suspect you would have to prove each one you have access to.
> 
> 
> 
> 

That's the thing I don't really remember what email is connected to it. But how do I contact support? Through the email? I have  the db url, db password and username. I just don't remember the email and password to the account 😭😭

**Reply by GaryAustin1** [reply]
Yes their email is contact for you.  Having no email you can prove is yours might be an issue.  Certainly having url and username are of no help as anyone could access that.
